optimum/intel/ipex/modeling_base.py

```diff
@@ -193,7 +193,7 @@ def forward(
             inputs["token_type_ids"] = token_type_ids
 
         outputs = self._call_model(**inputs)
-        return ModelOutput(**outputs) if isinstance(outputs, dict) else ModelOutput(logits=outputs[0])
+        return ModelOutput(**outputs) if isinstance(outputs, dict) else ModelOutput(last_hidden_state=outputs[0])
```

Comment:
it should be `logits` for all other classes (like `IPEXModelForSequenceClassification`, `IPEXModelForTokenClassification`, ...) can you update it accordingly ?

Reply:
Add a class parameter `output_name` would be more convenient and clean. Please take a review and let me know your opinion:)